Sources/UrlFormEncoding/UrlFormDecoder.swift

```diff
@@ -510,24 +526,71 @@ public final class UrlFormDecoder: Decoder {
     case custom((String) -> Date?)
   }
 
-  public struct ParsingStrategy {
-    let strategy: (String) -> [String: Any]
-
-    public static let accumulatePairs = custom { query in
-      var params: [String: Any] = [:]
-      for (name, value) in pairs(query) {
-        var values = params[name] as? [Any] ?? []
-        values.append(value)
-        params[name] = values
-      }
-      return params
-    }
-
+  public enum ParsingStrategy {
+    /// A parsing strategy that accumulates values when multiple keys are provided.
```

Comment:
amazing docs! we need to do this more after launch and use jazzy or something to generate docs